virbos-setup

```diff
@@ -209,15 +209,21 @@ getlayout() {
 				partitions="$partitions,$partition"
 			fi
 		done
-		ask_nodefault "What partition would you like to use as the boot partition?" "$partitions"
-		partition_boot=/dev/"$answer"
 		ask_yn "Do you want to install a bootloader?" y
-		[ "$answer" = y ] && use_grub=1
+		if [ "$answer" = 'y' ]; then
+			ask_nodefault "What partition would you like to use for the bootloader?" "$partitions"
+			partition_boot="$answer"
+		fi
```

Comment:
Please revert this change as there must be a partition for the kernel executable. It doesn't need to contain the bootloader, either.

Reply:
Oh sorry, that change is from before #3. Just replace those lines with what I added in my pr, sorry :grimacing: